README.md

```diff
@@ -10,15 +10,17 @@ You are only permitted to have *one* Commander. Commanders with Partner may be u
 
 ## Banlist
 
-_Correct as of 2019-03-13_ 
+_Correct as of 2019-03-14_ 
 
 - [Standard Commander ban list](https://scryfall.com/search?q=banned%3Aedh&order=name&as=grid&unique=cards)
+- Commanders with Keyword: "Partner" (You may play one of the partner commanders, but you may not have both in the command zone)
```

Comment:
Isn't this covered by line 9?

Reply:
I'd use this in favour of Line 9, or change Line 9 to reflect this. One of the instances is redundant, but additional clarity can't hurt.

Reply:
In that case I'd change this line to say `Multiple Commanders with Keyword: "Partner"` to better clarify that you can still run Commanders with Partner, just not two of them.

Reply:
- Multiple Commanders with the Keyword "Partner" You may still play cards with the "Partner" Keyword, however only one card with the "Partner" Keyword may be in your command zone

Proposed Change above, or is this just worse?

Reply:
Seems better, yeah!